client-frameworks-support/client-support-angular/projects/client-support-angular/src/lib/luigi-mock/luigi-mock.module.ts

```diff
@@ -23,155 +23,4 @@ import { LuigiMockEngine } from '@luigi-project/testing-utilities';
-                msg: 'luigi.ux.alert.hide',
-                data: event.data,
-                emulated: true
-              };
-              window.postMessage(response, '*');
-            },
-            'luigi.ux.set-current-locale': (event: any) => {
-              const response = {
-                msg: 'luigi.current-locale-changed',
-                currentLocale: event.data.data.currentLocale,
-                emulated: true
-              };
-              window.postMessage(response, '*');
-            },
-            // linkManager
-            'luigi.navigation.open': (event: any) => {
-              const response = {
-                msg: 'luigi.navigate.ok',
-                data: event.data,
-                emulated: true
-              };
-              window.postMessage(response, '*');
-            },
-            'luigi.navigation.splitview.close': (event: any) => {
-              const response = {
-                msg: 'luigi.navigate.ok',
-                data: event.data,
-                emulated: true
-              };
-              window.postMessage(response, '*');
-            },
-            'luigi.navigation.splitview.collapse': (event: any) => {
-              const response = {
-                msg: 'luigi.navigate.ok',
-                data: event.data,
-                emulated: true
-              };
-              window.postMessage(response, '*');
-            },
-            'luigi.navigation.splitview.expand': (event: any) => {
-              const response = {
-                msg: 'luigi.navigate.ok',
-                data: event.data,
-                emulated: true
-              };
-              window.postMessage(response, '*');
-            },
-            // storage
-            storage: () => {}
-          }
-        };
-
-        // Listen to the global 'message' event of the window object
-        window.addEventListener('message', (window as any).luigiMockEnvironment.msgListener);
-      }
-    };
-  }
-
-  /*
-   * This method takes a data object of type 'any' and vizualizes a simple container
-   * which holds data that is useful for e2e testing.
-   */
-  public static visualize(data: string): void {
-    let luigiVisualizationContainer: Element | null = document.querySelector('#luigi-debug-vis-cnt');
-    // Construct element structure if not already constructed
-    if (!luigiVisualizationContainer) {
-      luigiVisualizationContainer = document.createElement('div');
-      luigiVisualizationContainer.setAttribute('id', 'luigi-debug-vis-cnt');
-      // Hide the added DOM element to avoid interferring/overlapping with other elements during testing.
-      luigiVisualizationContainer.setAttribute('style', 'display:none');
-      document.body.appendChild(luigiVisualizationContainer);
-    }
-    const line: HTMLDivElement = document.createElement('div');
-    line.textContent = data;
-    luigiVisualizationContainer.appendChild(line);
-  }
-}
+export class LuigiMockModule { }
```

Comment:
I don't remember why this class was ever needed to begin with. Seems like MockEngine below does the same thing - has the same code. So why not delete this file altogether if it has nothing inside 🤔

Reply:
Currently it's not possible to delete the whole module because it loads [LuigiMockEngine.initPostMessageHook](https://github.com/SAP/luigi/pull/3654/files#diff-3c43c8edc309f76af69c1c9a1b7f9b82077d54090261cd3a123e1456b2999460L9)
Cannot move this "header" to testing-utilities because there is no angular available.

Reply:
...and it would be a breaking change when we change the loading mechanism of luigi mock engine